web/src/index.css

```diff
@@ -3,4 +3,7 @@
 :root {
   --link: var(--pink-4);
   --padding: var(--size-3);
+
+  --redwood: #bf4722;
+  --stripe: #635aff;
```

Comment:
Additions to our CSS lib for the subtitle. The styling could use more work but it's a start.

Reply:
Nice!!! 🚀 

Yeah the styling will always need a bit more work